addon/utils/promise-ajax.js

```diff
@@ -0,0 +1,7 @@
+export default (options) => {
+  return new Promise((resolve, reject) => {
```

Comment:
I don’t think we can rely on native Promise (unless we don’t support things like older IE and whatnot). We might need to add `import { Promise } from 'rsvp’`...

Reply:
Can rely on native Promise if we're only using this as a test helper? Or will people run the tests against IE? Not something I really know about.

Reply:
I'll raise my hand here. I build my tests against IE11.

Reply:
Thanks @jherdman. I'll use rsvp.